pola/rpc_api/openapi-v1.yaml

```diff
@@ -250,6 +236,20 @@ components:
                     official_url:
                       type: string
                       nullable: true
+                    all_company_brands:
+                      type: array
+                      items:
+                        type: object
+                        additionalProperties: false
+                        properties:
+                          name:
+                            type: string
+                          logotype_url:
+                            type: string
+                            nullable: true
```

Comment:
Możemy zwracać tylko obiekty z `logotype_url`? Z punktu widzenia klienta nie ma sensu taki obiekt. Jeśli będzie łatwiej to można w oddzielnym PRze

Reply:
Ok. Zrobimy na to oddzielne zadanie.